### PR TITLE
Many Transaction Long Running Test

### PR DESCRIPTION
**Pull request by brianjohnson5972**
## Change Description
Added a long running integration test to send a very large number of transfer transactions between accounts.
#8067

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: tests/nodeos_high_transaction_test.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+# This test sets up <-p> producing node(s) and <-n - -p>
+#   non-producing node(s). The non-producing node will be sent
+#   many transfers.  When it is complete it verifies that all
+#   of the transactions made it into blocks.
+#
+###############################################################
+
+Print=Utils.Print
+
+from core_symbol import CORE_SYMBOL
```
Comment (jgiszczak) [inline tests/nodeos_high_transaction_test.py]: Duplicate of above.

File: tests/nodeos_high_transaction_test.py [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+        lastBlockNum = blockNum
+
+    nextTime = time.perf_counter()
+    Print("Validating transfers took %s sec" % (nextTime - startTranferValidationTime))
+
+    delayedReportError = False
+    if len(missingTransactions) > 0:
+        verboseOutput = "Missing transaction information: [" if Utils.Debug else "Missing transaction ids: ["
+        first = True
+        for missingTrans in missingTransactions:
```
Comment (jgiszczak) [inline tests/nodeos_high_transaction_test.py]: `verboseOutput += ", ".join([missingTrans if Utils.Debug else missingTrans["newer_trans_id"] for missingTrans in missingTransactions])`

File: tests/nodeos_high_transaction_test.py [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+            first = False
+
+        verboseOutput += "]"
+        Utils.Print("ERROR: There are %d missing transactions.  %s" % (len(missingTransactions), verboseOutput))
+        delayedReportError = True
+
+    if len(transBlockOrderWeird) > 0:
+        verboseOutput = "Delayed transaction information: [" if Utils.Debug else "Delayed transaction ids: ["
+        first = True
+        for trans in transBlockOrderWeird:
```
Comment (jgiszczak) [inline tests/nodeos_high_transaction_test.py]: As above, string join of list comprehension.

File: tests/Cluster.py [hunk 4]
```diff
@@ -35,6 +35,39 @@ def isValid(policy):
...
+        if self.accounts is None:
+            Utils.errorExit("FAILURE - create keys")
+        accountNum = 0
+        for account in self.accounts:
+            Utils.Print("NamedAccounts Name for %d" % (accountNum))
+            account.name=self.setName(accountNum)
+            accountNum+=1
+
+    def setName(self, num):
+        retStr="test"
```
[the remark this answers is not in the document]
Comment (brianjohnson5972) [reply tests/Cluster.py]: only 1-5 is allowed for account_name

File: tests/Cluster.py [hunk 5]
```diff
@@ -35,6 +35,39 @@ def isValid(policy):
...
+        maxDigitVal=5
+        maxDigits=8
+        temp=num
+        while len(digits) < maxDigits:
+            digit=(num % maxDigitVal)+1
+            num=int(num/maxDigitVal)
+            digits.append(digit)
+
+        digits.reverse()
+        for digit in digits:
```
[the remark this answers is not in the document]
Comment (brianjohnson5972) [reply tests/Cluster.py]: ok

File: tests/nodeos_high_transaction_test.py (hunk 1 above)
Comment (brianjohnson5972) [reply tests/nodeos_high_transaction_test.py]: ok

File: tests/nodeos_high_transaction_test.py (hunk 2 above)
Comment (brianjohnson5972) [reply tests/nodeos_high_transaction_test.py]: ok

File: tests/nodeos_high_transaction_test.py (hunk 3 above)
Comment (brianjohnson5972) [reply tests/nodeos_high_transaction_test.py]: ok

**Pull request merged**